img.js

```diff
@@ -292,6 +292,7 @@ async function resizeImage(src, options = {}) {
         if(metadata.format !== "svg" || !options.svgAllowUpscale) {
           resizeOptions.withoutEnlargement = true;
         }
+        sharpInstance.rotate();
```

Comment:
This is never reached if `stat.width < metadata.width || (options.svgAllowUpscale && metadata.format === "svg"` is false.

Reply:
I think that's OK. The issue is caused when resizing only, so the code is reached for the same cases when the bug is happening.